sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAdministrationModelFactory.cs

```diff
@@ -3,14 +3,16 @@
 
 using System;
 using System.Collections.Generic;
+using Azure.Core;
 using Azure.Security.KeyVault.Administration.Models;
 
 namespace Azure.Security.KeyVault.Administration
 {
     /// <summary>
     /// A factory class which constructs model classes for mocking purposes.
     /// </summary>
-    public static class KeyVaultAdministrationModelFactory
+    [CodeGenType("AzureSecurityKeyVaultAdministrationModelFactory")]
+    public static partial class KeyVaultAdministrationModelFactory
```

Comment:
Just `AdministrationModelFactory`, which matches our other factory classes we already GA'd.

Reply:
This type was there already. Do you want me to rename it?

Reply:
Oh, wow, you're right. I didn't realize that (or just flat-out forgot). @christothes it seems to me that we should leave this name as-is since "AdministrationModelFactory" is rather vague, even if it fits the existing pattern. Agreed?

Reply:
Giving timing, we'll leave it as-is for now. If there's concern, we still have a little time to change it.